catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/model/Segment.js

```diff
@@ -451,13 +493,20 @@ define([
             }
             //update and add values
             _.each(fields, function (field) {
-                field.saveData(backingData,ebrimTypes);
+                if (field.get('type') === 'date') {
```

Comment:
Think this check should be moved to the Field.js validate method so the user will get some kind of indication the date is bad.